working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<p>Well done on returning two values! You got it very right! 👏</p><p> </p><p>Quick suggestion for improvement: again using built-in functions: <a href="https://docs.python.org/3/library/functions.html#min" target="_blank">min</a></p><p> and <a href="https://docs.python.org/3/library/functions.html#min" target="_blank">max</a></p><p> </p><p>Let me clarify that you've done right by "re-inventing the wheel"! That's the first step. My suggestions for improvements are just "excuses" to point you out something you might not know about already! :)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>